mono/utils/mono-qsort.h

```diff
@@ -0,0 +1,21 @@
+/**
+ * \file
+ */
+
+#ifndef __MONO_QSORT_H__
+#define __MONO_QSORT_H__
+
+#include <stdlib.h>
+
+static inline
+void
+mono_qsort (void* base, size_t num, size_t size, int (*compare)(const void*, const void*))
+{
+	g_assert (compare);
+	g_assert (size);
+	if (num < 2 || !size || !base)
```

Comment:
`!size` is redundant - you assert that size is non-zero on the previous line.

Reply:
Code as if assert might be disabled?